### PR TITLE
fix(recover_backup): use correct file ext to restore backupt on rhl

### PR DESCRIPTION
**Pull request by aleksbykov**
Wrong extention was used upon restore back files for rhl based systems.

Fixes: scylladb/scylladb#19777

Additional commit should fix the logic with skip condition. Because upon create tables, logic of skip condition is reversed, need to set false in skip contition field
related to https://github.com/scylladb/scylla-cluster-tests/commit/19577df303c2c3131029d8f936beab6bfd4e43d7

### Testing

- [ ]

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Comment by aleksbykov** [comment]
The latest issue left with failing test for cassandra stress:
```
Sleeping 2s...
Running READ with 1000 threads 10 minutes
INFO  [main] 2024-08-16 18:09:28,583 GuavaCompatibility.java:204 - Detected Guava >= 19 in the classpath, using modern compatibility layer
INFO  [main] 2024-08-16 18:09:28,668 Cluster.java:280 - DataStax Java driver 3.11.5.3 for Apache Cassandra
WARN  [main] 2024-08-16 18:09:28,846 SnappyCompressor.java:42 - Error loading Snappy library (java.lang.UnsatisfiedLinkError: /tmp/snappy-1.1.10-f52595de-1078-4845-929a-39ad15f59587-libsnappyjava.so: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /tmp/snappy-1.1.10-f52595de-1078-4845-929a-39ad15f59587-libsnappyjava.so)). Snappy compression will not be available for the protocol.
type       total ops,    op/s,    pk/s,   row/s,    mean,     med,     .95,     .99,    .999,     max,   time,   stderr, errors,  gc: #,  max ms,  sum ms,  sdv ms,      mb
Failed to connect over JMX; not collecting these stats
java.lang.IllegalStateException: The requested compression is not available (some compression require a JAR to be found in the classpath)
```
job: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/abykov/job/rolling-upgrade-staging-abykov/job/rolling-upgrade-centos9-test/7/

Looks like loader is not cofigured correctly and missing libld-linux-x86_64 file

File: sdcm/cluster.py [hunk 1]
```diff
@@ -5122,6 +5122,8 @@ def node_setup(self, node, verbose=False, **kwargs):  # pylint: disable=unused-a
         # Install java-8 for workaround hdrhistogram
         if node.distro.is_rhel_like:
             node.install_package(package_name='java-1.8.0-openjdk-devel')
+            node.install_package(package_name="libc6-compat")
+            node.install_package(package_name="gcompat")
```
Comment (roydahan) [inline sdcm/cluster.py:5126]: Don't we use docker based loaders?
In that case, maybe it's missing from the docker image?
But what has changed?

Comment (fruch) [reply sdcm/cluster.py:5126]: It's the new c-s images, and seems we are missing snappy

We should raise it and get Dusan to fix it

**Comment by fruch** [comment]
We can drop the last commit here, and get it merged

**Review by fruch — DISMISSED** [review]
LGTM

But we don't need the last commit here

**Comment by vponomaryov** [comment]
> LGTM
> 
> But we don't need the last commit here

Agree. @aleksbykov please, remove the last commit so we could merge the PR.

**Review by vponomaryov — APPROVED** [review]
LGTM

File: sdcm/cluster.py (hunk 1 above)
Comment (roydahan) [reply sdcm/cluster.py]: @CodeLieutenant do you have an issue for it?
If not, please create yourself one.

Comment (xtrey) [reply sdcm/cluster.py]: @CodeLieutenant please share the link to the issue.

Comment (xtrey) [reply sdcm/cluster.py]: @CodeLieutenant I suppose this is it https://github.com/scylladb/cassandra-stress/issues/13?

Comment (CodeLieutenant) [reply sdcm/cluster.py]: > @CodeLieutenant I suppose this is it [scylladb/cassandra-stress#13](https://github.com/scylladb/cassandra-stress/issues/13)?

Yes it was, the issue is fixed in the latest c-s release 3.13.0, I'll be making a pull request to SCT as soon as I'm done testing it.

**Pull request merged**